### DISCUSSION TITLE
Issue with cached views

### DISCUSSION THREAD

**Discussion by rcerljenko** [spatie/laravel-health#109] [General]
Hi @freekmurze ,

Running `php artisan view:cache` results with this:
```
Unable to locate a class or view for component [health-logo].
```

Any ideas?

**Comment by freekmurze** [comment]
No ideas, didn't encounter this myself. If you find the error, feel free to PR a fix 👍

**Reply by rcerljenko** [reply]
I'm trying for the past hour but no good... I've managed to figure out that vendor:publish also fails while trying to publish view components:
```
  Copying directory [...\vendor\spatie\laravel-health\resources\views] to [...\resources\views\vendor\health]  DONE
  File [...\database\migrations\2022_06_27_071042_create_health_tables.php] already exists .......................... SKIPPED
  Copying directory [...\vendor\spatie\laravel-health\resources\lang] to [...\resources\lang\vendor\health]  DONE
   ERROR  Can't locate path: <...\vendor\spatie\laravel-health\src\/../Components>.  <= THIS IS IT
```

**Reply by rcerljenko** [reply]
ok i've figured it out.. we use this package for blade minification:
https://github.com/HTMLMin/Laravel-HTMLMin

and it doesn't support view components yet... => https://github.com/HTMLMin/Laravel-HTMLMin/pull/144
